pkg/server/server.go

```diff
@@ -98,14 +106,16 @@ func (s *Server) receivePop(w http.ResponseWriter, _ *http.Request) {
 		}
 	}
 
+	w.Header().Set(contentType, contentTypeJSON)
+
 	if msg == nil {
-		w.WriteHeader(http.StatusNoContent)
+		if err := json.NewEncoder(w).Encode(&NilMessage{}); err != nil {
+			http.Error(w, err.Error(), http.StatusInternalServerError)
+		}
```

Comment:
isn't cheaper to do something like this?

```suggestion
		if err := w.Write([]byte("{}")); err != nil {
			http.Error(w, err.Error(), http.StatusInternalServerError)
		}
```